CONTRIBUTING.md

```diff
@@ -100,23 +107,17 @@ For running tests you will need [Docker](https://www.docker.com/products/docker-
   - For actions like click, hover use according functions from `scenarioHelper.js` and pass them as scenario options `actions` property.
     ```js
     const { scenario, hover } = require('../scenarioHelper');
-    module.exports = [
-      scenario('hover', { actions: [hover('.element-selector')] }),
-    ];
+    module.exports = [scenario('hover', { actions: [hover('.element-selector')] })];
```

Comment:
Why changed here?

Reply:
Ohh! I didn't make that change. Looks like it was the linter/formatter that runs after saving/committing the file.

### Before saving

![image](https://user-images.githubusercontent.com/45748283/180014519-ebd148a5-f0cb-4155-adfd-d824a21a4fa2.png)

### After saving

https://user-images.githubusercontent.com/45748283/180015109-28b8ed5f-b99c-4b8a-a8d2-c5011e71b10c.mp4

### After committing

Same as after saving

Reply:
**[DONE]** I saved the file without formatting as mentioned [below](https://github.com/iTwin/iTwinUI/pull/711#pullrequestreview-1046485696). Thus, we can ignore the changes due to formatting for now.